src/views/adminPage.ejs

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin Page</title>
+  <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+  <h1>Admin Page</h1>
+  <table>
+    <thead>
+      <tr>
+        <th>Album Title</th>
+        <th>Artist</th>
+        <th>Price</th>
+        <th>Select</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% function formatPrice(price) {
+          return price.toLocaleString('en-US', { style: 'currency', currency: 'USD' });
+      } %>
```

Comment:
Just a suggestion, but generally, I would define functions at the top of the EJS file, so it's easy to find them all and they're not interspersed with the html template code.